frontend/validator.html

```diff
@@ -15,26 +16,32 @@
       <form>
         <p>
           Validate URL:
-          <input id="input_url" type="url" value="https://status.crdmp.ch/">
-          <input id="submit_validate_url" type="submit" value="Validate URL">
+          <input id="input_url" type="url" value="https://status.crdmp.ch/" />
+          <input id="submit_validate_url" type="submit" value="Validate URL" />
         </p>
 
         <p>Paste your output here:</p>
-        <textarea id="content"></textarea>
+        <textarea id="validator_content"></textarea>
         <input id="submit_validate" type="submit" value="Validate">
       </form>
 
       <div id="banner">
-        <div id="OK">OK</div>
-        <div id="ERROR">ERROR</div>
-        <div id="processing">Processing request, please hold ...</div>
-        <div id="fetching">Fetching data, please hold ...</div>
+        <pre id="OK">OK</pre>
+        <pre id="ERROR">ERROR</pre>
+        <pre id="processing">Processing request, please hold ...</pre>
+        <pre id="fetching">Fetching data, please hold ...</pre>
```

Comment:
The capitalized values are sent by the server, the other events originate locally.